Wireframe/index.html

```diff
@@ -4,10 +4,46 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="./css/style.css">
   </head>
   <body>
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+
+    <header>
+      <h1>Git Made Easy</h1>
+      <h2>An explanation of the mysteries of Git.</h2>
+    </header>
+
+    <div class="main">
```

Comment:
Is there a semantic html element to identify the main region of the page?

What difference does it make, using div or some other html element?